keyboards/doio/kb04/readme.md

```diff
@@ -0,0 +1,27 @@
+# doio/kb04
+
+![doio/kb04](https://cdn.shopify.com/s/files/1/0463/5980/6101/products/4_f40396cc-ea72-47a5-887f-bbeeb4023b11_1100x.png?v=1654787686)
```

Comment:
Please rehost on Imgur.

Reply:
Imgur is not available to me (can't create an account). Have hosted it to Flickr instead.